catalogue/webapp/pages/works.tsx

```diff
@@ -253,7 +257,7 @@ const Works: NextPage<Props> = ({ works, worksRouteProps }) => {
 
 export const getServerSideProps = async (
   context: GetServerSidePropsContext
-) => {
+): Promise<GetServerSidePropsResult<Props>> => {
```

Comment:
For my own TS knowledge, is there a reason for works and images' getServerSideProps` to have different syntax/types?

Reply:
I don't think so, they should be consistent (and I suspect it's an accident that they're not) – I've tried to correct it in #8308, because this is still broken.

Reply:
I decided to do the types separately, because they were getting a bit complicated; once that's merged this PR should get much simpler.